src/App.tsx

```diff
@@ -1,24 +1,143 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useMemo, useEffect } from 'react';
+import debounce from 'lodash.debounce';
+import { TodoCreate } from './components/TodoCreate';
+import { TodoInfo } from './components/TodoInfo';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { client } from './utils/fetchClient';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
+import { url } from './components/url';
 
-const USER_ID = 0;
+export const USER_ID = '6757';
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>();
+  const [temporaryTodos, setTemporaryTodos] = useState<Todo[]>();
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+
+  const countNotComplited = useMemo(
+    () => todosFromServer?.some((todo: Todo) => !todo.completed),
+    [todosFromServer],
+  );
+
+  const countComplited = useMemo(
+    () => todosFromServer?.some((todo: Todo) => todo.completed),
+    [todosFromServer],
+  );
+
+  const fetchTodos = (pass: string, callback?: () => void): void => {
+    client
```

Comment:
you can change loading state here
```suggestion
  const fetchTodos = (pass: string, callback?: () => void): void => {
    setIsLoading(true);
    client
```